main.go

```diff
@@ -43,13 +44,18 @@ func init() {
 	flag.StringVar(&inPath, "in-file", "", "File path for the input certificate(s).")
 	flag.StringVar(&outPath, "out-file", "-", "File path for the output JSON.")
 	flag.StringVar(&outStat, "out-stat", "-", "File path for the output stats.")
+	flag.BoolVar(&prettyPrint, "pretty", false, "Use this flag to pretty print supported lints in JSON format")
```

Comment:
IMHO this should be `--list-lints-json` or something like that. JSON isn't pretty, and minified JSON certainly isn't pretty.